mne/viz/_3d.py

```diff
@@ -382,18 +426,13 @@ def plot_source_estimates(stc, subject=None, surface='inflated', hemi='lh',
     hemi : str, 'lh' | 'rh' | 'split' | 'both'
         The hemisphere to display. Using 'both' or 'split' requires
         PySurfer version 0.4 or above.
-    colormap : str
-        The type of colormap to use.
+    colormap : str | array
+        Name of colormap to use or a custom look up table (an n x 4
```

Comment:
@Eric89GXL, can you confirm that it's number of interpolation points for a mayavi. I've poked around in pysurfer, but can't be 100% sure.

FYI, I just copied this over from pysurfer docstring to improve the this one. I had gotten confused on why an array could be passed earlier before realizing the different types `colormap` could take on.